flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/PartitionProducerStateProvider.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.flink.runtime.jobgraph.IntermediateDataSetID;
```

Comment:
Can you expand commit message to include at least the same content as in the PR description?